telegram-bot-bumblebee-core/src/main/java/com/github/bumblebee/command/autocomplete/AutocompleteAddCommand.java

```diff
@@ -31,12 +31,12 @@ public void handleCommand(Update update, Long chatId, String argument) {
             return;
         }
 
-        if(!handler.addTemplate(argument.trim())){
-            botApi.sendMessage(chatId, "Wrong template, try again.");
+        if(argument==null || !handler.addTemplate(argument.trim())){
+            botApi.sendMessage(chatId, "wrong template, try again.");
```

Comment:
As you can see, you rewrote last commit with your changes. Even if so, could you please make these changes manually to not throw previous changes away?